Disable configure tests for the RT library, package tarball and latex
docs.

This patch is necessary with the dyninst-lite tar file for versions
of dyninst before the -DBUILD_DOCS=Off option.  The lite tar file
does not include the docs subdirs.


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b5a7c2..fd35d34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,10 @@ if(${SYMREADER} MATCHES symtabAPI)
 endif()
 
 #Build the RT library as a seperate project so we can change compilers
+
+message(STATUS "Skipping DyninstAPI_RT and RT library")
+if(FALSE)
+
 message(STATUS "Configuring DyninstAPI_RT")
 file(REMOVE_RECURSE ${RT_BINARY_DIR}/CMakeCache.txt ${RT_BINARY_DIR}/CMakeFiles ${RT_BINARY_DIR}/Makefile ${RT_BINARY_DIR}/Makefile)
 file(MAKE_DIRECTORY ${RT_BINARY_DIR})
@@ -118,9 +122,14 @@ add_custom_target(DyninstRT
 
 install(SCRIPT "${RT_BINARY_DIR}/cmake_install.cmake")
 
+endif()
+
 set (VERSION_STRING "${DYNINST_MAJOR_VERSION}.${DYNINST_MINOR_VERSION}.${DYNINST_PATCH_VERSION}")
 set (DYNINST_NAME "DyninstAPI-${VERSION_STRING}")
 
+message(STATUS "Skipping package tarballs and latex docs")
+if(FALSE)
+
 if(BUILD_TARBALLS)
 find_package(Git)
 if(GIT_FOUND)
@@ -197,6 +206,8 @@ else()
   message(STATUS "LaTeX not found")
 endif()
 
+endif()
+
 if(UNIX)
   message(STATUS "Adding Unix-specific dependencies")
   add_dependencies(dynDwarf libdwarf_imp)
